docs/DeveloperGuide.md

```diff
@@ -236,27 +236,34 @@ _{Explain here how the data archiving feature will be implemented}_
 
 ### User stories
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
 | Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
 | -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| `* * *`  | potential user                                  | save contacts of my suppliers and stores in same location | easily refer to the contacts and give them a call for updates                |
+| `* * *`  | potential user                                  | save products in the same location as suppliers and stores | easily refer keep track of stocks in various stores and their suppliers                 |
+| `* * *`  | new user                                       | access the command list/user guide               | easily refer to instructions for commands and usage instructions                                                                       |
+| `* * *`  | standard user                                       | add a new supplier/product/store contacts entry               | remove entries that I no longer need                                   |
```

Comment:
Oops! I think the benefit for this is wrong!

Also, I was wondering if we should combine the benefits stated in potential users and use the combined version for this user story?

Apart from this, everything LGTM!